R/board_url.R

```diff
@@ -304,11 +329,12 @@ http_download <- function(url, path_dir, path_file, ...,
     }
 
     headers <- httr::add_headers(
+      add_headers$headers,
```

Comment:
This is because as it is here, the `add_headers` object was created with `httr::add_headers()` and you can't put those together:

``` r
h <- httr::add_headers(a = 1, b = 2)
h
#> <request>
#> Headers:
#> * a: 1
#> * b: 2

httr::add_headers(h, c = 3)
#> Error in f(init, x[[i]]): is.request(y) is not TRUE
httr::add_headers(h$headers, c = 3)
#> <request>
#> Headers:
#> * a: 1
#> * b: 2
#> * c: 3
```

<sup>Created on 2023-04-14 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

I see what you're saying about having the argument use a named character instead and I'll check that out.